onnx/reference/ops/_op.py

```diff
@@ -167,3 +167,18 @@ def __init__(self, onnx_node: NodeProto, run_params: Dict[str, Any]):
                 self.axes = None
             elif isinstance(self.axes, list):
                 self.axes = tuple(self.axes)
+
+    def is_axes_empty(self, axes):
+        return axes is None or len(axes.shape) == 0 or axes.shape[0] == 0
+
+    def HandleAxes(self, axes):
```

Comment:
minor nit: I think the naming convention is `handle_axes` (same comment as Xavier made earlier for above method).